src/Lumper.CLI/Program.cs

```diff
@@ -3,9 +3,12 @@ namespace Lumper.CLI;
 using CommandLine;
 using Lib.BSP;
 using Lib.BSP.IO;
+using NLog;
 
 internal sealed class Program
 {
+    private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
     public static void Main(string[] args)
```

Comment:
ew past me made a main that returns void

Reply:
Hadn't touched in this PR but okay, added `1`/`0` exit codes

Reply:
there was a return added and thats how I saw it iirc